lib/experimental/new_embed.dart

```diff
@@ -559,6 +566,10 @@ major browsers, such as Firefox, Edge (dev channel), or Chrome.
       _loadAndShowGist(analyze: false);
     }
 
+    if (gistId.isEmpty) {
```

Comment:
Ultimately, we'll need this functionality for code loaded from a repo or the API docs as well. The ability to load from those sources hasn't been added to the new playground (something for which we should already have a tracking issue but don't, which is on me).

Reply:
#1346 and #1347 created.